src/amp.js

```diff
@@ -69,6 +69,9 @@ startupChunk(self.document, function initial() {
   installPerformanceService(self);
   /** @const {!./service/performance-impl.Performance} */
   const perf = Services.performanceFor(self);
+  if (self.document.hasAttribute('i-amphtml-no-boilerplate')) {
```

Comment:
shouldn't this be:
`if (self.document.documentElement.hasAttribute('i-amphtml-no-boilerplate')) {`

looks like it's missing `documentElement`.

Reply:
fixed. thanks!